storage-inventory/ops/README.md

```diff
@@ -128,139 +179,318 @@ The [`CADC Direct Data Service`](https://www.cadc-ccda.hia-iha.nrc-cnrc.gc.ca/en
-            - the `{local directory}` in `baseStorageDir` the is the location _on the container_ which the `baseURL` `/results` path will be mapped to.
+1. <a id='configuration-luskan'>**luskan**</a> - Query service
+
+    - Required for: Storage site and Global site
+    - Container image: Use the latest `storage-inventory/luskan` image from [`images.opencadc.org`](#resource-repository)
+    - Configuration notes:
+        - `luskan.properties`:
+            - `org.opencadc.luskan.isStorageSite` - for a storage site, this should be set to `true`.  The content of the inventory database is different between a storage site and a global site.
+            - `org.opencadc.luskan.allowedGroup` is an IVOA GMS group [resourceID](#term-resourceid).
+                - the GMS service must be configured in the available Registry service.
+        - `catalina.properties`:
+            - the `org.opencadc.luskan.uws.username` database account is generally the same as the 'TAP admin user' configured when creating the [database](#configuration-database).
+            - the `org.opencadc.luskan.tapadm.username` database account is the same 'TAP admin user'.
+            - the `org.opencadc.luskan.query.username` database account is the 'TAP query user' account.
+        - `cadc-tap-tmp.properties`:
+            - `org.opencadc.tap.tmp.TempStorageManager.baseURL` is the URL for _this_ `luskan` service, plus the `/results` path.
+                - the `{server name}` in the `baseURL` is the public name of the proxy fronting your `luskan` service (e.g. `www.example.org`).
+                - the `{luskan path}` is the path to your `luskan service` (likely just 'luskan', e.g. `www.example.org/luskan`).
+            - the above `/results` path will be mapped to the path _in the container_ specified by `org.opencadc.tap.tmp.TempStorageManager.baseStorageDir`.  Ideally, this path will be a file-system that is shared among all `luskan` instances for your site.
+    - See the [opencadc storage inventory luskan](https://github.com/opencadc/storage-inventory/tree/master/luskan) documentation for more configuration details.
     - test with, e.g., `curl https://www.example.org/luskan/availability`
 
-
-
-
-
-
+1. <a id='configuration-raven'>**raven**</a> - File location service
+
+    - Required for: Global site only
+    - Container image: Use the latest `storage-inventory/raven` image from [`images.opencadc.org`](#resource-repository)
+    - Configuration notes:
+        - `raven.properties`
+            - (optional) `org.opencadc.raven.publicKeyFile` and `org.opencadc.raven.privateKeyFile`:
+                - These are optional but needed so that raven can generate 'pre-authorized' URLs for files, reducing the number of authentication queries required for a user to download a file.  The authentication information is embedded in a specially encoded URL.
```

Comment:
```suggestion
                - These are optional optimizations needed so that raven can generate 'pre-authorized' URLs for files, allowing the minocs that serve the file to skip this step before delivering the file.  The authentication information is embedded in a specially encoded URL.
```